optimizer_wrapper.rb

```diff
@@ -190,8 +190,6 @@ def self.solve(service_vrp, job = nil, block = nil)
 
     tic = Time.now
 
-    Interpreters::MultiTrips.new.expand(vrp)
-
```

Comment:
(c/p) May be in the future we can move this logic out of Interpreters::MultiTrips into vrpclass because it is just transformation of the input. However while this logic is in the class Interpreters::MultiTrips`, I think it is better to keep it in the old location (unless there is a specific reason to not to) because all other expands are inside solve at the moment.